test/functional/wallet_importdescriptors.py

```diff
@@ -480,7 +480,9 @@ def run_test(self):
         addr = wmulti_pub.getnewaddress('', 'bech32')
         assert_equal(addr, 'bcrt1qp8s25ckjl7gr6x2q3dx3tn2pytwp05upkjztk6ey857tt50r5aeqn6mvr9') # Derived at m/84'/0'/0'/1
         change_addr = wmulti_pub.getrawchangeaddress('bech32')
-        assert_equal(change_addr, 'bcrt1qt9uhe3a9hnq7vajl7a094z4s3crm9ttf8zw3f5v9gr2nyd7e3lnsy44n8e')
+        assert_equal(change_addr, 'bcrt1qzxl0qz2t88kljdnkzg4n4gapr6kte26390gttrg79x66nt4p04fssj53nl')
+        assert(any([x == send_txid for x in self.nodes[0].getrawmempool(True)]))
+        assert(any([x['txid'] == send_txid for x in wmulti_pub.listunspent(0)]))
```

Comment:
```suggestion
        assert(any(x == send_txid for x in self.nodes[0].getrawmempool(True)))
        assert(any(x['txid'] == send_txid for x in wmulti_pub.listunspent(0)))
```

No benefit to making it a list - just causes every element to be compared (using the generator directly stops comparing as soon as it finds a match)

Or even better and clearer...

```suggestion
        assert(send_txid in self.nodes[0].getrawmempool(True))
        assert(send_txid in (x['txid'] for x in wmulti_pub.listunspent(0)))
```

Reply:
done